pkg/controller/job/job_controller.go

```diff
@@ -848,6 +849,19 @@ func (jm *Controller) syncJob(ctx context.Context, key string) (forget bool, rEr
 					job.Status.StartTime = &now
 				}
 			}
+			podFailureCondChanged := false
+			if manageJobErr != nil {
+				if action == metrics.JobSyncActionPodsCreated {
```

Comment:
That's very specific and only temporary case we don't want to report, because as soon as the SA gets created the job will work just fine.